js/test/Ice/optional/InitialI.ts

```diff
@@ -10,11 +10,11 @@ export class InitialI extends Test.Initial {
         current.adapter.getCommunicator().shutdown();
     }
 
-    pingPong(obj: Ice.Nullable<Ice.Value>, current: Ice.Current): Ice.Nullable<Ice.Value> {
+    pingPong(obj: Ice.Value | undefined, current: Ice.Current): Ice.Value | undefined {
```

Comment:
Maybe this change is intentional, but just wanted to point out that this
`Nullable` was changed to `| undefined` instead of `| null`.